samples/config.default.js

```diff
@@ -52,23 +52,23 @@ exports.credentials.conversation = {
 // Watson Vision Recognition
 // https://www.ibm.com/watson/developercloud/text-to-speech.html
 exports.credentials.visual_recognition = {
-    key: '',
-    version: ''
+    key: "",
+    version: "v1"
 }
 
 // Watson Tone Analyzer
 // https://www.ibm.com/watson/developercloud/tone-analyzer.html
 exports.credentials.tone_analyzer = {
-    password: '',
-    username: ''
+    password: "",
+    username: ""
 };
 
 // Twitter
 exports.credentials.twitter = {
```

Comment:
Yeah .. all twitter reference has been removed from the lib itself tjbot.js.
We can also remove it from here too.